src/central-server/admin-service/core-api/src/main/java/org/niis/xroad/cs/admin/api/domain/ManagementRequestView.java

```diff
@@ -49,11 +50,14 @@ public class ManagementRequestView {
     private String securityServerOwnerName;
     private String xroadInstance;
     private String memberCode;
-
     private String memberClass;
-
     private String serverCode;
-
+    private String clientOwnerName;
```

Comment:
map to XRoadId types where it is possible. This will give us flexibility to quickly get encodedId or separate values